.github/workflows/ci.yaml

```diff
@@ -0,0 +1,139 @@
+name: CI
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Setup Node.js 20
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: npm
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+  test_nodejs:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build
+    - run: npm test run
+
+  test_deno:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: denoland/setup-deno@v1
+      with:
+        deno-version: v1.x
+    - run: |
+        deno eval '
+        import { parse } from "./src/web-csv-toolbox.ts";
+
+        const csv = `name,age
+        Alice,42
+        Bob,69`;
+
+        for await (const record of parse(csv)) {
+          console.log(record);
+        }'
```

Comment:
You make a valid point, but since we have not developed the project so that code executed in Deno can be mixed with code executed in Node.js, please allow me to reflect that point at a later date.

Resolve.